src/test/java/step1/SetTest.java

```diff
@@ -0,0 +1,52 @@
+package step1;
+
+import org.assertj.core.api.Assertions;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.CsvSource;
+import org.junit.jupiter.params.provider.ValueSource;
+
+import java.util.HashSet;
+import java.util.Set;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+class SetTest {
+
+    private Set<Integer> numbers;
+
+    @BeforeEach
+    void setUp() {
+        numbers = new HashSet<>();
+        numbers.add(1);
+        numbers.add(1);
+        numbers.add(2);
+        numbers.add(3);
+    }
+
+    @Test
+    void sizeReturnsSizeOfSet() {
+        Integer actual = numbers.size();
+        Integer expected = 3;
+        assertThat(actual).isEqualTo(expected);
+    }
+
+    @ParameterizedTest
+    @ValueSource(ints = { 1, 2, 3 })
+    void containsReturnTrueWhenGivenElementIsInSet(Integer source) {
+        Boolean actual = numbers.contains(source);
+        assertThat(actual).isTrue();
+    }
+
+    @ParameterizedTest
+    @CsvSource(value = {"1, true", "2, true", "3, true", "4, false", "5, false"})
+    void containsReturnBooleanByWhetherGivenElementIsInSetOrNot(String sourceString, String expectedString) {
```

Comment:
다음과 같이 실제로 사용할 타입을 지정해도 정상적으로 테스트가 동작합니다. 단순 참고차 말씀드려봅니다.
```suggestion
    void containsReturnBooleanByWhetherGivenElementIsInSetOrNot(int source, boolean expected) {
```

Reply:
오 감사합니다!